nservicebus/operations/installers.md

```diff
@@ -15,6 +15,8 @@ Installers ensure that endpoint specific artifacts (e.g. queues, directories, da
 
 ## Running installers
 
+NOTE: Running installers requires the process to run with permissions to create/delete tables, queues, folders and other resources. We recommend to apply the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege) and prevent running an endpoint process with too many permissions. This means only running the installers during setup with elevated permissions and not while processing messages.
```

Comment:
Like this version much better 👍 I think the last sentence could use some tweaking, how about something like this:
"Consider running the installers only during the initial setup using elevated permissions"?